### PR TITLE
Translations R-es.po

### PR DESCRIPTION
**Pull request by yabellini**
I did the remaining translations to Spanish.  Thanks so much, @zkamvar and @joelnitta, for building this.

Thanks a lot, @acrall, for working so hard to advance the localization of The Carpentries' materials.

### CODE REVIEW

File: po/R-es.po [hunk 1]
```diff
@@ -218,23 +221,23 @@ msgid ""
 "The actual schedule may vary slightly depending on the topics and exercises "
 "chosen by the instructor."
 msgstr "El horario real puede variar ligeramente dependiendo de los temas y "
-"ejercicios elegidos por el instructor."
+"ejercicios elegidos por la instructora."
```
Comment (ErinBecker) [inline po/R-es.po:224]: :-)

Comment (zkamvar) [inline po/R-es.po:224]: I'm really happy to see this one!

Comment (yabellini) [reply po/R-es.po:224]: Using the feminine as default is part of the agreements the Carpentries Spanish-speaking community made.  I'm only applying those.  :-)

**Comment by zkamvar** [comment]
Thank you, @yabellini!

**Pull request merged**